apps/common/src/python/mediawords/solr/request.py

```diff
@@ -219,21 +270,38 @@ def solr_request(path: str,
 
         content_encoded = content.encode('utf-8', errors='replace')
 
-        request = Request(method='POST', url=abs_url)
-        request.set_header(name='Content-Type', value=content_type)
-        request.set_header(name='Content-Length', value=str(len(content_encoded)))
-        request.set_content(content_encoded)
-
+        results = []
+        for collection in collections:
+            abs_uri = furl(f"{solr_url}/{collection}/{path}")
+            abs_uri = abs_uri.set(params)
+            abs_url = str(abs_uri)
+            request = Request(method='POST', url=abs_url)
```

Comment:
We can drop the `abs_url = str(abs_uri)` unless the `abs_url` is used somewhere else.
```
 request = Request(method='POST', url=str(abs_url))
```